### PR TITLE
Clone of TNW web page

### PR DESCRIPTION
**Pull request by Zappat0n**
This is a pull request for merging the TheNextWeb clone to the main branch.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+}
+
+.search-icon {
+  top: -6px;
+  right: 52px;
+}
+
+/* <!-- MAIN SECTION --> */
+
+#main-section {
```
Comment (uryelah) [inline styles.css]: - [x] Avoid using id selectors to style elements in CSS. [Check this article with reasons for why that's a bac practice](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) and change the selectors to class selectors(you can always [combine selectors to be have more specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) if that was the reason for using ids).

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Rashad Rashad & Angel Barros Pazos**, I'm Sarah your reviewer for this PR 👩‍🚀 

You guys did a great job building a very responsive clone of the Next Web page and nailing your semantic HTML. There are some minor issues to fix to do with the CSS code style and how some sections of the page look on larger screens but you are really close to the finish line so fear not!

Read the review comments below carefully and take some time to compare your clone with the [original page](https://thenextweb.com/), using your browser responsiveness tools, and then get to coding!

If you think you need a refresher about how to use responsiveness tools I'd check [this Firefox, the best browser for developers, turorial](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode).

Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below before requesting the next code review**


- [x] Avoid using id as style selectors in CSS, prefer class selectors, check the line comment below this message.

- [x] Improve the footer at medium to large screens in such a way that it utilizes the same shades of grey that the original one does, has similar type fonts to the original under the icons, and that the bottom area text is all in the same line, not broken in three lines.
If you are not sure how to get the same shade of grey the original site uses you can just use the [developer tools inspector of your browser](https://developers.google.com/web/tools/chrome-devtools/css) and get the colors from the element styles or use a [color picker like the one Firefox has](https://developer.mozilla.org/en-US/docs/Tools/Eyedropper)
- [x] Make sure that on large screens the sections below the main articles at wide screens all take the same width and are aligned horizontally like on the original page:

**Compare the origin site on larger screens and your clone in the screenshots below:**

 <table style="width:100%">
  <tr>
    <th>Original Page</th>
    <th>Clone</th>
  </tr>
  <tr>
    <td valign="top"></td>
    <td></td>
  </tr>
<tr>
<td>
  
</td>
<td>
  
</td>
</tr>
</table> 

---

- [x] [OPTIONAL] Let's add some box-shadow under the fixed top navbar to make it look closer to the original one and make it 'pop' in the eye by giving it some depth.

- [x] [OPTIONAL] Why don't you add an icon to the tab of your clone to make the page more professional? Check [this StackOverflow thread about it if you are not sure how to achieve it.](https://stackoverflow.com/questions/4888377/how-to-add-a-browser-tab-icon-favicon-for-a-website)

- [x] [OPTIONAL] A lot of the links in the original page have [hover effects](https://css-tricks.com/almanac/selectors/h/hover/) that change the text color or add underlines, why don't you play a bit with hover effects and transitions? That really improves the page UX(user experience) and makes it feel more 'real'.

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: styles.css (hunk 1 above)
Comment (od-c0d3r) [reply styles.css:64]: Thank you @uryelah 

Very Professional, descriptive, and clean code review ⭐⭐⭐📝

We are working on it and would push it ASAP...

@od-c0d3r
@Zappat0n

**Comment by o-rashad** [comment]
Thank you @uryelah 

Very professional, descriptive, and oriented Code review ⭐️⭐️⭐️📝

We are currently working on fixing it and would be pushing it ASAP...

@Zappat0n , @od-c0d3r

**Comment by o-rashad** [comment]
### **UPDATE after 1st Code Review.**

**Based on the last code review instructions
we finished the below :**

1- IDs selectors in CSS file has been changed to classes selectors.
2- Footer section has been improved.
3- Hover with transtion effects has been implemented.
4- Latest News section overall margins has been improved.
5- Shadow effect has been added to the top Navbar.

Thanks to @uryelah

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi team :wave: 

Good job, go on with your next task.

Just an observation, you missed adding the logo for screen size 1024px. As you scroll down through the page the logo should appear.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Comment by o-rashad** [comment]
Good Catch 🕵. @uche-inyama 

We gonna try to fix it push it and merge the pull request. 📝

Thank you for the CR 🙋‍♂️ @uche-inyama , @uryelah ...

**Pull request merged**